modules/snooker/src/test/ts/browser/PasteColumnOperationsTest.ts

```diff
@@ -498,6 +498,72 @@ UnitTest.test('PasteColumnOperationsTest', () => {
     TableOperations.pasteColsAfter, 0, 0, 1
   );
 
+  Assertions.checkPasteRaw(
```

Comment:
Would these be better in `modules/snooker/src/test/ts/browser/PasteCellsOperationsTest.ts`?

Also wondering if it would be worth adding a test that causes new columns and row to be created e.g. pasting into the bottom right of the table.

Reply:
Thanks for this, agree on both fronts - have made the changes